irodsinfo/views.py

```diff
@@ -22,6 +22,45 @@
 logger = logging.getLogger(__name__)
 
 
+class IrodsConfigMixin:
+    """Mixin for iRODS configuration views"""
+
+    @staticmethod
+    def create_env_config(user, irods_backend) -> dict:
```

Comment:
Good work refactoring. I would name this method `get_irods_client_env()`.

Also, we do not use type hints in SODAR, as they kind of have to be used everywhere to work in a meaningful way. One day we'll do a huge refactoring and add them everywhere :)